docs/UserGuide.md

```diff
@@ -435,12 +434,12 @@ e.g. `Apex Legends` may be stored as `ApexLegends`.
 </div>
 
 Examples:
-* `game --add Valorant` Adds a brand-new game into the game list with the **unique** `GAME_ID` 'Valorant'.
+* `game --add RocketLeague` Adds a brand-new game into the game list with the **unique** `GAME_ID` 'RocketLeague'.
```

Comment:
All other references and mentions of the game RocketLeague in the captions are in `code`. Can perhaps convert this too.
'RocketLeague' -> `RocketLeague`

Reply:
I think for the other commands, the name of the game or the friend's ID is also in '....'. If want to update to `code` have to update the rest as well.

Example in `game --delete`:
<img width="658" alt="Screenshot 2021-11-07 at 1 42 33 PM" src="https://user-images.githubusercontent.com/61085398/140633942-6a7305c2-0e00-4968-b88b-1aa62cece109.png">

Reply:
Interesting. It appears that in the examples the gameId and friendId are in 'single quotes' but in the image captions, the gameId and friendId are in `code` .